spk/sonarr/src/service-setup.sh

```diff
@@ -9,20 +9,15 @@ SPK_SONARR="${SYNOPKG_PKGINST_TEMP_DIR}/share/NzbDrone/NzbDrone.exe"
 CONFIG_DIR="${SYNOPKG_PKGDEST}/.config"
 PID_FILE="${CONFIG_DIR}/NzbDrone/nzbdrone.pid"
 
-GROUP="sc-media"
+# Sonarr uses the home directory to store it's ".config"
+# But there is a problem:
+# On DSM5 this is: PKG/var/.config/
+# On DSM6 this is: PKG/.config/
+# So we have to do upgrade magic
```

Comment:
I do not like such magic. Isn't it possible to set `HOME` to `PKG/var/` in process environment at Sonarr startup with `env` for instance ? it is already done for other packages...